analyzer.go

```diff
@@ -543,8 +543,7 @@ func (gosec *Analyzer) ParseErrors(pkg *packages.Package) error {
 // AppendError appends an error to the file errors
 func (gosec *Analyzer) AppendError(file string, err error) {
 	// Do not report the error for empty packages (e.g. files excluded from build with a tag)
-	r := regexp.MustCompile(`no buildable Go source files in`)
-	if r.MatchString(err.Error()) {
+	if strings.Contains(err.Error(), "no buildable Go source files in") {
 		return
 	}
```

Comment:
The test is indeed faster

But checking the error message still looks dirty

Is there a case where this could not be used ?

https://pkg.go.dev/go/build#NoGoError

Reply:
@alexandear ?

Reply:
Thanks. Created https://github.com/securego/gosec/pull/1273